source/index.html.md

```diff
@@ -36,106 +36,64 @@ Run the following command in a terminal emulator to see a live JSON response fro
-    },
-  ...
-```
-
-The chaptered content of this piece of guidance is split into multiple pages and the API response is structured so that these pages, or content resources,
-can be visited at a distinct path.
-
-#### Metadata fields
-
-While the `details` fields of the API response may differ per format, the _metadata_ fields, those at the top level of the JSON response, are generally very similar as they describe the content resource, eg. when the item was last updated.
+This endpoint responds with a resource containing all the information needed to display a web page on GOV.UK.
 
-The [reference documentation](reference.html#contentitem) contains much more detail on each of the fields in the API response.
+#### The `GET /content/{path}` response
+
+The reference documentation contains a [full example of a response](https://alphagov.github.io/govuk-content-api-docs/reference.html#example-responses) from the `GET` content by `{path}` endpoint.
+
+Broadly speaking the response contains 3 different types of data concerning the resource:
+
+  1. _Metadata_ about the resource, such as when it was published and what locale the information corresponds to:
+
+    ```json
+     "title" : "Businesses and charging VAT",
+     "public_updated_at" : "2014-12-19T15:27:25Z",
+     "base_path" : "/vat-businesses",
+     "locale" : "en",
+     "api_path" : "/api/content/vat-businesses"
+    ```
+
+  2. Contextual links to other related content resources contained within the response:
+
+    ```json
+      "links" : {
+          "mainstream_browse_pages" : [
+             {
+                "public_updated_at" : "2015-06-24T13:56:39Z",
+                "base_path" : "/browse/tax/vat",
+                "api_path" : "/api/content/browse/tax/vat",
+                "locale" : "en",
+                ...
+                "description" : "Includes online returns, rates, charging and record keeping",
+                "title" : "VAT",
+             }
+          ],
+          "parent" : [
+             {
+                "title" : "VAT",
+                "description" : "Includes online returns, rates, charging and record keeping",
+                "links" : {
+                   "parent" : [
+                      {
+                         "public_updated_at" : "2015-04-08T10:48:40Z",
+                         "api_path" : "/api/content/browse/tax",
+                         "locale" : "en",
+                         ...
+                         "base_path" : "/browse/tax",
+                         "title" : "Money and tax",
+                         "description" : "Includes VAT, debt and inheritance tax"
+                      }
+              ...
+    ```
+
+  3. The content itself:
+
+    ```json
+      "details": {
+        "body" : "\n<div class=\"highlight-answer\">\n<p>The standard <abbr title=\"Value Added Tax\">VAT</abbr> rate is <em>20%</em></p>\n</div>..."
+        ...
+      }
+    ```
```

Comment:
This information looks really useful but I think it probably belongs in reference documentation.